Customer.java

```diff
@@ -4,7 +4,7 @@
 
 public class Customer {
 	
-	private ArrayList<Account> accounts;
+	public ArrayList<Account> accounts;
```

Comment:
Consider using the mutators and accessors rather than making the properties public

Reply:
Moreover, I believe there is _getAccounts_ method that you can use in such a case.